module/os/linux/spl/spl-kmem.c

```diff
@@ -26,7 +26,6 @@
 #include <sys/sysmacros.h>
 #include <sys/kmem.h>
 #include <sys/vmem.h>
-#include <linux/mm.h>
```

Comment:
The file still uses `is_vmalloc_addr()` and trying to backport the patch to 0.8.x results in compile errors. It turns out sys/kmem.h includes linux/mm.h but only in git master.

Is this indirection intentional?